Tori.teng/cancel rehydration (#13476)
DataDog/documentation: content/en/logs/log_configuration/rehydrating.md

{{< img src="logs/archives/log_archives_historical_index_selector.png" alt="Log Explorer" width="75%">}}

> ### Canceling ongoing historical views

Reviewer: ```suggestion
### Cancel ongoing historical views
```

> Cancel ongoing Rehydrations directly in the Rehydrate from Archives page to avoid starting Rehydrations with the incorrect time range or when you accidentally make typos in your indexing query.

Reviewer: ```suggestion
Cancel ongoing Rehydrations directly in the Rehydrate from Archives page to avoid starting Rehydrations with the incorrect time range or when there are typos in your indexing query.
```

> The logs already indexed will remain queryable until the end of the retention period selected for that historical view, and all the logs already scanned and indexed will still be billed. 

Reviewer: ```suggestion
The logs already indexed remain queryable until the end of the retention period that was selected for that historical view. All logs already scanned and indexed are still billed. 
```

> {{< img src="logs/archives/log_archives_cancel_ongoing_rehydration.png" alt="Canceling Ongoing Rehydrations" width="75%" >}}

Reviewer: ```suggestion
{{< img src="logs/archives/log_archives_cancel_ongoing_rehydration.png" alt="The Rehydrate from Archives page showing a historical view in calculating status and the cancel rehydration option" width="75%" >}}
```
